### PR TITLE
Projects Model

### PR DESCRIPTION
**Pull request by gideonthomas**

### CODE REVIEW

**Comment by alicoding** [comment]
Travis is failing on linting here.

File: app/scienceapi/projects/migrations/0002_auto_20160418_1954.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```
Comment (alicoding) [inline app/scienceapi/projects/migrations/0002_auto_20160418_1954.py]: This file is auto generated right?

**Comment by gideonthomas** [comment]
ugh

File: app/scienceapi/projects/models.py [hunk 2]
```diff
@@ -1,3 +1,90 @@
-# from django.db import models
+from django.db import models
+from django.utils import timezone
```
Comment (alicoding) [inline app/scienceapi/projects/models.py]: You're not using this.

File: app/scienceapi/projects/models.py [hunk 3]
```diff
@@ -1,3 +1,90 @@
...
+    project_url = models.URLField(
+        max_length=500,
+        null=True,
+        help_text='URL to the project\'s landing page on the web',
+    )
+    slug = models.SlugField(
+        max_length=150,
+        null=True,
+        verbose_name='Mozilla Science URL Slug',
+        help_text='Slug appended to the Mozilla Science project\'s '
```
Comment (alicoding) [inline app/scienceapi/projects/models.py]: I love this `help_text` \o/

**Comment by alicoding** [comment]
I guess this patch is just the model, right?

File: app/scienceapi/projects/migrations/0002_auto_20160418_1954.py (hunk 1 above)
Comment (gideonthomas) [inline app/scienceapi/projects/migrations/0002_auto_20160418_1954.py]: yep

**Comment by gideonthomas** [comment]
yeah model*s*

**Comment by gideonthomas** [comment]
Finally!!!!
@alicoding re-r?

**Comment by alicoding** [comment]
R+!!!

**Comment by gideonthomas** [comment]
ty

**Comment by gideonthomas** [comment]
damnit, forgot to squash

**Pull request merged**